index.html

```diff
@@ -176,16 +182,7 @@
         </dt>
         <dd>
           The <a>Document</a> is loaded in the prerender mode and is not yet
-          visible. User agent support of the <code>prerender</code> return
-          value of the <a>visibilityState</a> attribute is OPTIONAL.
-        </dd>
-        <dt>
-          <dfn id="idl-def-.unloaded">unloaded</dfn>
-        </dt>
-        <dd>
-          The user agent is to <a>unload</a> the <a>Document</a>. User agent
-          support of the <code>unloaded</code> return value of the
-          <a>visibilityState</a> attribute is OPTIONAL.
```

Comment:
Sure. I'm not opposing to deprecating it, but let's document the discussion, implementation status, usage (if any), etc.